Automattic/wp-calypso
# Ribbon: Move to calypso-ui

This is a leaf component that doesn't have any counterpart in `@wordpress/components`. It might come in handy in Jetpack (to highlight paid Gutenberg blocks in the block picker).

#### Changes proposed in this Pull Request

Ribbon: Move to calypso-ui

#### Testing instructions

```
npm run distclean && npm ci
npm start
```

Verify that the ribbon still works:

- http://calypso.localhost:3000/devdocs/design/ribbon
- http://calypso.localhost:3000/themes
